3_javascript/javascript3/week2/CurrencyCalculator/script.js

```diff
@@ -0,0 +1,240 @@
+domListener();
+function domListener() {
+  document.addEventListener("DOMContentLoaded", domLoader);
+}
+
+function domLoader() {
+  const inputAmount = document.getElementById("amount");
+  const clickConvert = document.getElementById("convert");
+
+  clickConvert.addEventListener("click", getData);
+  inputAmount.addEventListener("input", () => (amount = inputAmount.value));
+
+  const fromCurrencySelect = document.getElementById("fromCurrency");
+  fromCurrencySelect.addEventListener(
+    "input",
+    () => (fromCurrency = fromCurrencySelect.value)
+  );
+
+  const toCurrencySelect = document.getElementById("toCurrency");
+  toCurrencySelect.addEventListener(
+    "input",
+    () => (toCurrency = toCurrencySelect.value)
+  );
+
+  const apiKey = "1bd1c44ffdf9e866b3f41d32";
+  let allData = "";
+  let fromCurrency = "EUR";
+  let toCurrency = "DKK";
+  let amount = 0;
+
+  const currencies = [
```

Comment:
You already have a list of all the currencies in the response of the API, so you could build this currency array from the API there. Visit this link for reference [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

Reply:
Thanks for the suggestion!  I made the change taking the currencies from the API